.luacheckrc

```diff
@@ -0,0 +1,73 @@
+std             = "ngx_lua"
+unused_args     = false
+redefined       = false
+max_line_length = false
+
+
+globals = {
+    "_KONG",
+    "kong",
+    "ngx.IS_CLI",
+}
```

Comment:
Just a nit, but it'd be good to not have kong-specific stuff in here so that nobody accidentally commits code using kong globals.

```suggestion
globals = {
    "ngx.IS_CLI",
}
```